content/intro-to-storybook/react/en/conclusion.md

```diff
@@ -21,6 +21,10 @@ Want to dive deeper? Here are helpful resources.
 
 - [**Visual Testing Handbook**](https://www.chromaticqa.com/book/visual-testing-handbook) dives deep into using Storybook to visual test components. Free 31-page ebook.
 
+- [**Storybook Discord**](https://discord.gg/UUt2PJb) will put you in touch with the Storybook community. Not only offers a means to help you but for you to help the community also.
```

Comment:
```suggestion
- [**Storybook Discord chat**](https://discord.gg/UUt2PJb) puts you in contact with the Storybook community. Get and give help to other Storybook users.
```

Reply:
@domyen marking this as solved per latest commit i just did